content/en/blog/_posts/2023-11-1-Gateway-API-Future.md

```diff
@@ -0,0 +1,257 @@
+---
+layout: blog
+title: "New Experimental Features in Gateway API v1.0"
+date: 2023-11-02T10:00:00-08:00
+slug: gateway-api-ga
+---
+
+***Authors:*** John Howard (Google), Simone Rodigari (IBM), Candace Holman (Red Hat), Gaurav K Ghildiyal (Google), Dave Protasowski (VMware)
+
+Recently, the [Gateway API](https://gateway-api.sigs.k8s.io/) [announced its v1.0 GA release](/blog/2023/10/31/gateway-api-ga/), marking a huge milestone for the project.
+
+Along with stabilizing some of the core functionality in the API, a number of exciting new *experimental* functionality has been added.
+
+## Backend TLS Policy
+
+`BackendTLSPolicy` is a new Gateway API type used for specifying the TLS configuration of the connection from the Gateway to backend Pods via the Service API object.
+It is specified as a [Direct PolicyAttachment](https://gateway-api.sigs.k8s.io/geps/gep-713/#direct-policy-attachment) without defaults or overrides, applied to a Service that accesses a backend, where the BackendTLSPolicy resides in the same namespace as the Service to which it is applied.
+One can use the same BackendTLSPolicy for all the different Routes that might point to the referenced Service.
+
+While there were existing ways provided for [TLS to be configured for edge and passthrough termination](https://gateway-api.sigs.k8s.io/guides/tls/#tls-configuration), this new API object specifically addresses the configuration of TLS in order to convey HTTPS from the Gateway dataplane to the backend.
+This is referred to as "backend TLS termination" and enables the Gateway to know how to connect to a backend Pod that has its own certificate.
+
+![Termination Types](https://gateway-api.sigs.k8s.io/geps/images/1897-TLStermtypes.png) 
+
+The specification of a BackendTLSPolicy consists of:
+- TargetRef - Defines the targeted API object of the policy.  Only Service is allowed.
```

Comment:
It looks like we should be using code style for all field names in this blog post, so `targetRef` here, and similar changes throughout: https://kubernetes.io/docs/contribute/style/style-guide/#use-code-style-for-object-field-names-and-namespaces.